File: tests/end-to-end/efs/python/stack.py
from aws_cdk import Stack
import aws_cdk as cdk
import aws_cdk.aws_efs as efs
from constructs import Construct

class EfsStack(Stack):
  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
    super().__init__(scope, construct_id, **kwargs)

    # Resources
    fileSystem = efs.CfnFileSystem(self, 'FileSystem',
        )

File: tests/end-to-end/efs/csharp/Stack.cs
using Amazon.CDK;
using Amazon.CDK.AWS.EFS;
using Constructs;
using System.Collections.Generic;

namespace EfsStack
{
    public class EfsStackProps : StackProps
    {
    }

    public class EfsStack : Stack
    {
        public EfsStack(Construct scope, string id, EfsStackProps props = null) : base(scope, id, props)
        {

            // Resources
            var fileSystem = new CfnFileSystem(this, "FileSystem", new CfnFileSystemProps
            {
            });
        }
    }
}
